### PR TITLE
Remove proxy on login due to timeout errors, remove SSL v1.3 stuff an…

### PR DESCRIPTION
**Pull request by gtracer**
…d remove debug logs

### CODE REVIEW

File: src/truecallerpy/login.py [hunk 1]
```diff
@@ -73,7 +72,7 @@ async def login(phone_number: str) -> dict:
         "clientsecret": "lvc22mp3l1sfv6ujg83rd17btt",
     }
     try:
-        async with httpx.AsyncClient(proxies=PROXY, verify=SSL_CONTEXT) as client:
+        async with httpx.AsyncClient() as client:
```
Comment (eduardodatalane) [inline src/truecallerpy/login.py]: I may not have the context on this, but why are we not removing the PROXY from verify_otp in this PR as well? Can you add context to the PR if there's a reason for it?

Comment (gtracer) [reply src/truecallerpy/login.py]: There isn't. We should and I forgot to. Will fix

Comment (gtracer) [reply src/truecallerpy/login.py]: Good catch

Comment (gtracer) [reply src/truecallerpy/login.py]: Ok I made an update - I decided to keep the cache but remove the SSL stuff 

In a day or two we can see if the proxy is causing the timeout issues I saw. If it is I will remove the proxy and we can use VPN

**Review by eduardodatalane — APPROVED** [review]

**Pull request merged**